src/preprocessing/passes/foreign_theory_rewrite.cpp

```diff
@@ -24,12 +24,113 @@ namespace preprocessing {
 namespace passes {
 
 using namespace CVC4::theory;
-
 ForeignTheoryRewrite::ForeignTheoryRewrite(PreprocessingPassContext* preprocContext)
     : PreprocessingPass(preprocContext, "foreign-theory-rewrite"),
       d_cache(preprocContext->getUserContext()){};
 
-Node ForeignTheoryRewrite::simplify(Node n) { assert(false); }
+Node ForeignTheoryRewrite::simplify(Node n)
+{
+  vector<Node> toVisit;
```

Comment:
```suggestion
  std::vector<Node> toVisit;
```

Not sure where we have a `using namespace std;` but either we should have it in this file or we should just write it out (which I prefer). Same applies to other `std` objects.